packages/govuk-frontend/src/govuk/components/header/header.mjs

```diff
@@ -83,23 +84,42 @@ export class Header extends GOVUKFrontendComponent {
     this.$menu = $menu
     this.$menuButton = $menuButton
 
-    // Set the matchMedia to the govuk-frontend desktop breakpoint
-    this.mql = window.matchMedia('(min-width: 48.0625em)')
+    this.setupResponsiveChecks()
+
+    this.$menuButton.addEventListener('click', () =>
+      this.handleMenuButtonClick()
+    )
+  }
+
+  /**
+   * Setup viewport resize check
+   *
+   * @private
+   */
+  setupResponsiveChecks() {
+    const breakpoint = getBreakpoint('desktop')
+
+    if (!breakpoint.value) {
+      throw new ElementError({
+        componentName: 'Header',
+        identifier: `CSS custom property (\`${breakpoint.property}\`) on pseudo-class :root`
```

Comment:
**nitpick** I think there's a couple of backticks missing.

```suggestion
        identifier: `CSS custom property (\`${breakpoint.property}\`) on pseudo-class \`:root\``
```

Reply:
Yeah, left them off but will add 👍

Is it just types that we don't wrap in backticks?

```console
Radios: Root element (`$module`) is not of type HTMLElement
```